README.md

```diff
@@ -108,6 +108,7 @@ after `--`, for instance you could run `pnpm test -- --concurrency 2`.
   checks, as well as whatever other code checks any packages support.
 - `pnpm fix`: Run auto fixes, including reformatting code and auto-fixing lint
   rules where possible.
+- `pnpm turbo build`: Build all packages in the monorepo using turborepo.
```

Comment:
let's remove this line, there shouldn't usually be a good reason to build all packages in the monorepo with turbo -- instead we should be doing things like linting / testing / starting / etc and let turbo build what needs to be built.

Reply:
cc @jayantk I'm not sure how we should be updating Devin's PRs....

Reply:
meta comment is i do think a bit more docs here would be useful. based on these directions, I don't know how to run scripts or anything. Last time i worked with contract manager i spent some time digging around to figure this out.